lib/gridstore/Cargo.toml

```diff
@@ -34,6 +34,7 @@ common = { path = "../common/common" }
 
 # this is not on dev-dependencies because dev-dependencies cannot be optional :(
 rocksdb = { version = "0.23.0", optional = true }
+priority-queue = "2.5.0"
```

Comment:
Naive question, can we use the native binary heap instead?

https://doc.rust-lang.org/std/collections/struct.BinaryHeap.html

Reply:
No, because we need to be able to update any gaps, not just the top one. Particularly when flushing, the top region might not be the same as the one which must free some gaps.

BinaryHeap and PriorityQueue are different because the latter lets you access and modify any item, while the first works best when we are only interested in the top one.

Reply:
That makes a lot of sense. Thanks for elaborating